src/styles/blocks/_page.scss

```diff
@@ -0,0 +1,11 @@
+html {
```

Comment:
Move the 'html' and 'body' styles to a more general file, like '_base.scss', since they are not specific to the catalog page layout.